frontend/src/Frontend/UI/Widgets.hs

```diff
@@ -1111,8 +1099,8 @@ uiInputWithPopover body getStateBorderTarget mkMsg cfg = divClass "popover" $ do
 
   _ <- runWithReplace (divClass "popover__message" blank) $ leftmost
     [ popoverDiv . popoverToAttrs <$> onMsg
-    , popoverDiv popoverHiddenAttrs <$ onBlur
-    , popoverDiv . popoverToAttrs <$> current dPopState <@ onFocus
+    -- , popoverDiv popoverHiddenAttrs <$ onBlur
```

Comment:
Leftovers?

Reply:
They are required, I'll uncomment them. That's a debugging leftover because otherwise it's really difficult to debug the styling.